src/View/Parsers/ShortcodeParser.php

```diff
@@ -733,6 +733,6 @@ function ($matches) use ($tags, $parser) {
 
         $this->extend('onAfterParse', $content);
 
-        return $content;
+        return $content ?? '';
```

Comment:
This was needed to respect the typehinting of methods that call this and return its value.
It's also just sensible.